lib/orderbook/OrderBook.ts

```diff
@@ -391,14 +392,19 @@ class OrderBook extends EventEmitter {
    * @param quantityToRemove the quantity to remove from the order, if undefined then the full order is removed
    * @returns `true` if the order or portion thereof was removed, otherwise `false`
    */
-  private removePeerOrder = (orderId: string, pairId: string, quantityToRemove?: number) => {
+  private removePeerOrder = (orderId: string, pairId: string, peerPubKey: string, quantityToRemove?: number) => {
     const matchingEngine = this.getMatchingEngine(pairId);
     try {
-      const removeResult = matchingEngine.removePeerOrder(orderId, quantityToRemove);
+      const removeResult = matchingEngine.removePeerOrder(orderId, peerPubKey, quantityToRemove);
       this.emit('peerOrder.invalidation', { orderId, pairId, quantity: removeResult.order.quantity });
       return true;
     } catch (err) {
-      this.logger.error(`attempted to remove non-existing orderId (${orderId})`);
+      let errMsg = `attempted to remove non-existing orderId (${orderId})`;
```

Comment:
we don't know anymore that the orderId doesn't exists. just that it doesn't exist for this specific peer